packages/selection/src/containers/SelectionContainer.spec.js

```diff
@@ -0,0 +1,493 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import KEY_CODES from '../utils/KEY_CODES';
+import SelectionContainer from './SelectionContainer';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+
+jest.useFakeTimers();
+
+describe('SelectionContainer', () => {
+  const itemValues = ['Item-1', 'Item-2', 'Item-3'];
+  let wrapper;
+
+  const BasicExample = ({ direction, defaultFocusedIndex, selectedAriaKey } = {}) => (
+    <SelectionContainer direction={direction} defaultFocusedIndex={defaultFocusedIndex}>
+      {({ getContainerProps, getItemProps, focusedItem, selectedItem }) => (
+        <div {...getContainerProps({ 'data-test-id': 'container' })}>
+          {itemValues.map(item => {
+            const ref = React.createRef();
+            const isSelected = item === selectedItem;
+            const isFocused = item === focusedItem;
+
+            return (
+              <div
+                {...getItemProps(
+                  {
+                    key: item,
+                    item,
+                    focusRef: ref,
+                    ref,
+                    selectedAriaKey,
+                    'data-test-id': 'item',
+                    'data-focused': isFocused,
+                    'data-selected': isSelected
+                  },
+                  {
+                    selectedAriaKey
+                  }
+                )}
+              >
+                {item}
+              </div>
+            );
+          })}
+        </div>
+      )}
+    </SelectionContainer>
+  );
+
+  const findItems = enzymeWrapper => enzymeWrapper.find('[data-test-id="item"]');
+  const findContainer = enzymeWrapper => enzymeWrapper.find('[data-test-id="container"]');
+
+  beforeEach(() => {
+    act(() => {
```

Comment:
`act` is a new utility that will flush any `useEffect` calls on first render so we can test them correctly.